src/basic/log.c

```diff
@@ -1345,6 +1349,14 @@ void log_received_signal(int level, const struct signalfd_siginfo *si) {
                          signal_to_string(si->ssi_signo));
 }
 
+void set_log_syntax_callback(log_syntax_callback_t cb, void *p) {
```

Comment:
Here too, it doesn't matter *too* much, but `userdata` would be immediately clear, while `p` isn't.